### DISCUSSION TITLE
sensor.epex_spot_at_net_price

### DISCUSSION THREAD

**Discussion by aichingerk** [mampfes/ha_epex_spot#69] [General]
Hi,
I look under developer tools the states from the sensor.epex_spot_at_net_price
The time is now: 22:14 and the state of the sensor state is 9.078

When i scroll down on the attributes to the current time between 22:00 - 23:00, the state on the attribute is 5.0508

  - start_time: "2023-11-23T21:00:00+00:00"
    end_time: "2023-11-23T22:00:00+00:00"
    price_ct_per_kwh: 9.078
  - start_time: "2023-11-23T22:00:00+00:00"
    end_time: "2023-11-23T23:00:00+00:00"
    price_ct_per_kwh: 5.0508

can you tell me, why is there an offset off an -1 hour between attribute on the sensor and the display of the sensor ?

**Comment by mampfes** [comment]
In the developer tools, the timestamps are annotated in UTC time (as indicated by +00:00). Central European Time is 1h ahead of UTC, therefore the difference.
Home Assistant knowns your local time zone and will adjust all time stamps accordingly.
